### PR TITLE
Package upgrades

### PR DESCRIPTION
**Pull request by vader1986**

### CODE REVIEW

**Comment by turowicz** [comment]
KubernetesClient shouldn't be private as your package needs it to work right?

**Comment by turowicz** [comment]
KubernetesClient shouldn't be private as your package needs it to work right? @vader1986 @galvesribeiro

**Comment by vader1986** [comment]
Yes, sorry my mistake

File: src/Orleans.Clustering.Kubernetes/Orleans.Clustering.Kubernetes.csproj [hunk 1]
```diff
@@ -27,7 +27,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="KubernetesClient" Version="10.0.31" PrivateAssets="All" />
+    <PackageReference Include="KubernetesClient" Version="10.0.31" />
```
Comment (galvesribeiro) [inline src/Orleans.Clustering.Kubernetes/Orleans.Clustering.Kubernetes.csproj:30]: image

I think we should upgrade this as well. Not sure if there are too many breaking changes...

Comment (vader1986) [reply src/Orleans.Clustering.Kubernetes/Orleans.Clustering.Kubernetes.csproj:30]: There was this PR, that's why I didn't upgrade:
https://github.com/OrleansContrib/Orleans.Clustering.Kubernetes/pull/69

But makes sense to me. Will include that upgrade here.

**Comment by turowicz** [comment]
LGTM?

**Review by galvesribeiro — APPROVED** [review]

**Pull request merged**